### ISSUE TITLE
Check case studies for mentions of Docker and update as needed

### ISSUE THREAD

**Issue by celestehorgan** [kubernetes/website#30934]
Umbrella issue: https://github.com/kubernetes/website/issues/30771

## Description

Check [Case Studies](https://kubernetes.io/case-studies/) and for mentions of Docker. 

## Mitigation - case studies

We will not update case studies. For case studies that mention Docker, add a `warning` shortcode **to the bottom of the page**: 

```
{{< warning >}}
This case study mentions Docker, which is no longer compatible with Kubernetes. For more information, see the [Dockershim Deprecation FAQ](https://kubernetes.io/blog/2020/12/02/dockershim-faq/).
{{< /warning >}}
```

You may need to use the underlying HTML instead: 

```html
<div class="alert alert-danger warning callout" role="alert">
  <strong>Warning:</strong> This case study mentions Docker, which is no longer compatible with Kubernetes. For more information, see the <a href="https://kubernetes.io/blog/2020/12/02/dockershim-faq/" target="newwindow">Dockershim Deprecation FAQ</a>.
</div>
```

**Comment by nate-double-u** [comment]
Thanks for putting the list together @shannonxtreme

**Comment by nate-double-u** [comment]
I think that these 4 are borderline cases that may need a warning attached:
* https://kubernetes.io/case-studies/peardeck/index.html
* https://kubernetes.io/case-studies/prowise/index.html
* https://kubernetes.io/case-studies/squarespace/index.html
* https://kubernetes.io/case-studies/wink/index.html

But I'm not convinced that they become warning worthy with the dockershim removal. Thoughts?

**Comment by sftim** [comment]
https://kubernetes.io/case-studies/peardeck/index.html
 ➤ I'd reword “Docker containers” to “containers”

https://kubernetes.io/case-studies/prowise/index.html
 ➤ I'd reword “Dockerize” to “containize”

https://kubernetes.io/case-studies/squarespace/index.html
 ➤ I'd reword “Docker file” to “`Dockerfile`”

https://kubernetes.io/case-studies/wink/index.html
 ➤  I'd change “just over two years ago” to “in 2015”
 ➤ I'd also change “Dockerized” to “containerized”

_separately_, maybe we should omit older case studies from the index of case studes - and when we serve those old case studies, do so with a note about heritage? I do think that's a separate issue of course.

**Comment by celestehorgan** [comment]
@sftim I'm checking in on the case studies omitting thing, but I suspect the answer is "probably not". R
